static/app/views/eventsV2/table/quickContext.spec.tsx

```diff
@@ -315,6 +317,22 @@ describe('Quick Context', function () {
 
       userEvent.hover(screen.getByText('Text from Child'));
 
+      expect(await screen.findByText(/Events/i)).toBeInTheDocument();
+      expect(screen.getByText(/2.5m/i)).toBeInTheDocument();
+      expect(screen.getByText(/Users/i)).toBeInTheDocument();
+      expect(screen.getByText(/64k/i)).toBeInTheDocument();
+    });
+
+    it('Renders event and user counts', async () => {
```

Comment:
Should these assertions live in this "Renders event and user counts test"? Looks like you might've updated the wrong test

Reply:
yup you're right haha, will push the fix.